tests/conftest.py

```diff
@@ -57,6 +57,33 @@ def forecast_national(db_session):
 
     return None
 
+@pytest.fixture
+def forecast_national_ecmwf_and_xg(db_session):
+    t0_datetime_utc = (datetime.now(tz=timezone.utc) + timedelta(days=2)).replace(minute=0, second=0, microsecond=0)
+    # time detal of 2 days is used as fake forecast are made 2 days in the past,
+    # this makes them for now
+    # create
+    model_names_ecmwf_and_xg = ["pvnet_ecmwf", "National_xg"]
+    for i in range(len(model_names_ecmwf_and_xg)):
+        model_name = model_names_ecmwf_and_xg[i]
```

Comment:
Could simplify this to
```suggestion
    for model_name in model_names_ecmwf_and_xg:
```